src/rocker/nvidia_extension.py

```diff
@@ -123,7 +124,7 @@ def get_snippet(self, cliargs):
         return em.expand(snippet, self.get_environment_subs(cliargs))
 
     def get_docker_args(self, cliargs):
-        if get_docker_version() >= Version("19.03"):
+        if get_docker_version() >= Version("19.03") and platform.uname().processor != 'aarch64':
```

Comment:
My fault - it's more about Jetson issue. Here is more about it: [[1](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/l4t-base)], [[2](https://developer.nvidia.com/embedded/learn/tutorials/jetson-container)], [[3](https://forums.developer.nvidia.com/t/getting-gpu-docker-passthrough-working/220284/7)]. I don't know how it may looks with different platforms. Maybe it is a good idea to let users `force` type of nvidia command regardless of docker version validation?

Reply:
Yes, I think that enabling the force on this might make sense. I see that they use this older syntax. Is maybe our version check not quite right? Do you know what version of docker is running on the Jetson? Maybe a higher 19.x series?